apps/shared/lib/req_custom_cache.ex

```diff
@@ -1,7 +1,11 @@
-defmodule CustomCache do
+defmodule Transport.Shared.ReqCustomCache do
```

Comment:
Pour l'instant, ce cache est conçu pour être utilisé uniquement en développement.

Il a été crucial pour accélérer les itérations de développement tout en travaillant avec le "vrai volume", vu le nombre de ressources traitées.